### ISSUE TITLE
When Alice funds alpha asset...

### ISSUE THREAD

**Issue by D4nte** [comit-network/comit-rs#383]
When Alice funds the alpha asset, both Alice and Bob transition to the `AlphaFunded` state.
Correct actions be returned from now on.

Expected actions to be returned in GET swap:
- Alice: None (refund is handled in a separate issue)
- Bob: fund

Returning the correct actions once in `AlphaFunded` state and returning the correct information for `fund` action is part of this ticket.

DoD:
- [x] Write e2e test BTC->ETH

Notes:
Actions to be displayed are only for `fund beta`, `refund` action display tracked with #384 

Child of #405.
Child of #408.
Blocked by #380.
Blocks #385.
